apps/prairielearn/src/pages/instructorCourseAdminSettings/instructorCourseAdminSettings.html.ts

```diff
@@ -18,111 +18,174 @@ export function InstructorCourseAdminSettings({
       </head>
       <body>
         ${renderEjs(__filename, "<%- include('../partials/navbar'); %>", { ...resLocals })}
-        <main id="content" class="container-fluid">
+        <main id="content" class="container mb-4">
```

Comment:
Micro-nit: for consistency with the aforementioned pages, move this class to `<div class="card">`.

Reply:
Moved this class to the card div in [ad5d7a8](https://github.com/PrairieLearn/PrairieLearn/pull/9386/commits/ad5d7a8bae4c6acd3db8daddaa0e356b0b47ee2b).